src/lib/common/watchdog.c

```diff
@@ -5,10 +5,12 @@
 #include "task.h"
 #include "iwdg.h"
 #include "task_priorities.h"
+// #include "memfault/ports/watchdog.h"
```

Comment:
I left these commented out for now in case we ever want to include them again. I can remove them if that is cleaner. Or maybe I could add some comments.